db/migrate/20250119125736_add_absent_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddAbsentToUsers < ActiveRecord::Migration[7.2]
+  def change
+    add_column :users, :absent, :boolean, default: false, null: false
```

Comment:
should we also update `email` field to add default as it is optional for now?

Reply:
I would prefer it being `nil` if not filled in.